draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2607,15 +2614,39 @@ Internet-Draft                 ACTN POI                       July 2023
    update the bandwidth information of the multi-layer single-domain IP
    link supported by the updated Ethernet bundled link.
 
-5.2.3. Multi-layer SRLG Configuration
+5.2.3. Multi-layer TE path properties Configuration
+
+   The MDSC can discover the TE path properties (e.g., the list of
+   SRLGs, the delay) of a multi-layer IP link from the TE path
+   properties of the IP routers terminating the IP link (e.g., the list
+   of SRLGs reported by the P-PNC using the packet TE topology model),
+   of the optical path (e.g., the list of SRLGs reported by the O-PNC
```

Comment:
Can we separate this long sentence with period? 
The MDSC can discover the TE path properties (e.g., the list of
SRLGs, the delay) of a multi-layer IP link from the TE path
properties of the IP routers terminating the IP link (e.g., the list
   of SRLGs reported by the P-PNC using the packet TE topology model).
Moreover .....

Reply:
Difficult to split: rephrased as a bullet list